UserInfo/README.md

```diff
@@ -34,6 +34,16 @@ returns:
 
 	value: uint256 - user default public key, can be 0.
 
+`getSigningBox` - allows DeBot to receive handle to keypair which can be used to sign external messages to the address returned by `UserInfo.getAccount`. This function can return zero handle. In that case DeBot should not use it to sign messages and should implement other ways to recevie signing keypair (e.g using `SigningBoxHandle` interface or delegate DEngine to receive signing box from browser).
+
+arguments:
+
+    answerId: uint32 - function id of callback.
+
+returns:
+
+	value: uint32 - handle to signing keypair, can be 0.
```

Comment:
I would suggest to rename the variable to `handle` to make it more consistent with `SiningBoxInput` interface
```suggestion
	handle: uint32 - handle to signing keypair, can be 0.
```